circleguard/enums.py

```diff
@@ -369,6 +375,45 @@ def __init__(self, max_angle=10, min_distance=8):
         self.max_angle = max_angle
         self.min_distance = min_distance
 
+class CleanMode():
+    """
+    The specification of the options used to clean replays for comparison.
+
+    Parameters
+    ----------
+    value: int
+        One (or a bitwise combination) of :data:`~.CleanMode.VALIDATE`,
+        :data:`~.CleanMode.SYNCHRONIZE`, :data:`~.CleanMode.ALIGN`, :data:`~.CleanMode.SEARCH`.
+        The options used.
+    search_step: int
+        The time interval used when searching.
+    step_limit: int
+        The maximal amount of steps performed when searching.
+    """
+
+    VALIDATE = 1 << 0
+    SYNCHRONIZE = 1 << 1
+    ALIGN = 1 << 2
+    SEARCH = 1 << 3
+    ALL = VALIDATE + SYNCHRONIZE + ALIGN + SEARCH
```

Comment:
1-2 sentence comment about each of these would be nice. I know there's a lengthier explanation in each of their methods (`filter_single`, `align_clocks`, `align_coordinates`, etc) but I don't want to have to hunt those methods down to know at a glance what an option does.